### PR TITLE
squiggly line remover in the controllers 🙄

### PR DESCRIPTION
**Pull request by awqfrxx**
bro i hate squiggly lines, fixed return types and parameters

also re-added URL and some text to tell users to add https://

### CODE REVIEW

**Review by avanheeren — APPROVED** [review]
the devil works hard but Quinten works harder

**Comment by Koreyoshi159** [comment]
Like why? those lines make it a directory that is basically the same as /.

**Comment by awqfrxx** [comment]
> Like why? those lines make it a directory that is basically the same as /.

because i was annoyed

File: app/Http/Controllers/ActivityController.php [hunk 1]
```diff
@@ -78,16 +83,17 @@ public function update(Request $request, Activity $activity)
...
+     * @throws Exception
      */
     public function destroy(Activity $activity)
     {
         $activity->delete();
         return redirect(route('admin.index'));
     }
 
-    public function getValidate(Request $request)
+    public function getValidate(Request $request): array
```
Comment (AlexGit123) [inline app/Http/Controllers/ActivityController.php:96]: why'd you use : array here?

Comment (awqfrxx) [reply app/Http/Controllers/ActivityController.php:96]: return type for the function ig

**Review by Sanyi0001 — APPROVED** [review]
I don't mind them so discuss it with Hao:)

**Review by AlexGit123 — APPROVED** [review]

**Review by koenhamstra — APPROVED** [review]
Looks good

**Pull request merged**